Sources/SimpleCurrencyConverter/DependencyInjection/DependencyContainer.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+public final class DependencyContainer: DependencyProtocol {
```

Comment:
@ts-dzmitry-kurski Does `DependencyContainer` still needs to be public?

Reply:
@malayli No, it can be internal by default. Private is not an option because I created an instance of the `DependencyContainer` object inside `SimpleCurrencyConverter` class (I already checked it). So I can change the access level to internal. What do you think about it?

Reply:
@ts-dzmitry-kurski Ok please make it `internal`.